nixos/tests/babeld.nix

```diff
@@ -120,6 +120,9 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : {
     ''
       start_all()
 
+      client.systemctl("start network-online.target")
+      local_router.systemctl("start network-online.target")
+      remote_router.systemctl("start network-online.target")
       client.wait_for_unit("network-online.target")
       local_router.wait_for_unit("network-online.target")
       remote_router.wait_for_unit("network-online.target")
```

Comment:
Waiting for `network-online.target` is entirely redundant here, since the test waits for routing to converge further down.

```suggestion
```

Reply:
applied and confirmed it runs